fzf-marks.plugin.zsh

```diff
@@ -18,7 +18,7 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
-command -v fzf >/dev/null 2>&1 || return
+type -P fzf >/dev/null || return
```

Comment:
Zsh doesn't have `type -P`, but Zsh's `type -p` is actually closer to Bash's `type -P` (than to Bash's `type -p`).

```suggestion
type -p fzf >/dev/null || return
```